PyPoE/cli/exporter/wiki/parsers/passives.py

```diff
@@ -289,6 +290,10 @@ def export(self, parsed_args, passives):
                         data['icon'] = '%s (%s)' % (icon[-1], icon[-2])
                 else:
                     data['icon'] = icon[-1]
+            #atlas_start_node doesn't have an icon path
+            else:
+                data['icon'] = ''
+                warnings.warn('Icon path file not found for {}: {}'.format(passive['Id'], passive['Name']))
```

Comment:
Same comment as https://github.com/Project-Path-of-Exile-Wiki/PyPoE/pull/37#discussion_r744102703